core/src/main/java/com/linecorp/armeria/common/logging/LoggingDecoratorBuilder.java

```diff
@@ -53,6 +59,23 @@
     private Function<? super Throwable, ?> responseCauseSanitizer = DEFAULT_CAUSE_SANITIZER;
     private Function<? super HttpHeaders, ?> responseTrailersSanitizer = DEFAULT_HEADERS_SANITIZER;
 
+    /**
+     * Sets the {@link Logger} to use when logging.
+     * If unset, will use default logger that each logging implementation has.
+     */
+    public T logger(Logger logger) {
+        this.logger = requireNonNull(logger, "logger");
+        return self();
+    }
+
+    /**
+     * Returns the nullable {@link Logger} to use when logging.
```

Comment:
Just `Returns the {@link Logger} the user specified to use, or {@code null} if not set and a default logger should be used.`

Reply:
fixed